src/main/java/com/coveros/selenified/utilities/Listener.java

```diff
@@ -22,11 +22,9 @@
 
 import com.coveros.selenified.Browser;
 import com.coveros.selenified.exceptions.InvalidHubException;
-import com.coveros.selenified.services.HTTP;
-import com.coveros.selenified.services.Request;
 import com.coveros.selenified.utilities.Reporter.Success;
-import com.google.gson.JsonArray;
-import com.google.gson.JsonObject;
+import com.saucelabs.saucerest.SauceException;
+import com.saucelabs.saucerest.SauceREST;
 import org.testng.ITestResult;
 import org.testng.SkipException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.testng.SkipException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)